qa-pipelines/tasks/run-test.sh

```diff
@@ -178,7 +178,7 @@ if [[ ${TEST_NAME} == "acceptance-tests" ]] && [[ $pod_status -gt 0 ]]; then
 fi
 
 if [[ ${TEST_NAME} == "acceptance-tests" ]]; then
-    if [[ -n ${recurring_failures} ]]; then
+    if [[ ${CATS_RERUN} -eq 5 ]] && [[ -n ${recurring_failures} ]]; then
```

Comment:
FWIW, you can have defaults for variables; for example, `${recurring_failures:-}` makes an empty string the default.

Reply:
I was originally thinking that recurring_failures could also be `"unset"` when it got here, but after looking at it some more, I don't think that can happen. So maybe that's the easier solution.